template/assets/js/modules/dashboard.js

```diff
@@ -3,3 +3,6 @@
 * assets/js/modules/dashboard.js
 * TODO : build module; add functions (overview, view, add, update & delete) to let things happen
 */
+function mainDashboard(){
+
```

Comment:
Waarom is dit een lege functie???

Reply:
functie wordt in de toekomst aangeroepen door module dashboard, is nu alvast toegevoegd..